config.js

```diff
@@ -50,6 +51,7 @@ var environment = {
   awsRegion: process.env.AWS_REGION,
   sendgridApiKey: process.env.SENDGRID_API_KEY,
   sendgridFrom: process.env.SENDGRID_FROM,
+  gdalTranslateBin: process.env.GDAL_TRANSLATE_BIN,
```

Comment:
I'd say that is better to have the path specified. In some situations may not be easy to modify the `PATH` and in this case we actively throw an error if not set, making it easy to know what the problem is.